src/test/resources/application-unittest.yml

```diff
@@ -0,0 +1,35 @@
+spring:
+
+  # Spring data rest
```

Comment:
Die Datei `application-no-security.yml` kann aus dem internen Repo isi-internal entfernt werden, da diese nun im github-Repo vorhanden ist.

Die Datei `application-no-security.yml` kann auch aus isi-internal entfernt werden. Jedoch sollte die Datei von `src/test/resources/application-no-security.yml` nach `src/main/resources/application-no-security.yml` verschoben werden, da das Profil `no-security` auch während der lokalen Entwicklung verwendet wird.

Reply:
isi-backend/application-unittest.yml aus isi-internal gelöscht (da jetzt hier im Repo).

die Konfiguration aus application-no-security.yml in application-unittest.yml verschoben und appliation-no-security.yml gelöscht.

Reply:
die Änderungen noch pushen.

Reply:
bereits gepusht